lib/ansible/modules/source_control/gitlab_group.py

```diff
@@ -15,59 +15,69 @@
 module: gitlab_group
 short_description: Creates/updates/deletes Gitlab Groups
 description:
-   - When the group does not exist in Gitlab, it will be created.
-   - When the group does exist and state=absent, the group will be deleted.
-   - As of Ansible version 2.7, this module make use of a different python module and thus some arguments are deprecated.
+  - When the group does not exist in Gitlab, it will be created.
+  - When the group does exist and state=absent, the group will be deleted.
 version_added: "2.1"
 author: "Werner Dijkerman (@dj-wasabi)"
 requirements:
-    - python-gitlab python module
+  - python-gitlab python module
 options:
-    server_url:
-        description:
-            - Url of Gitlab server, with protocol (http or https).
-        required: true
-    validate_certs:
-        description:
-            - When using https if SSL certificate needs to be verified.
-        type: bool
-        default: 'yes'
-        aliases:
-            - verify_ssl
-    login_user:
-        description:
-            - Gitlab user name.
-    login_password:
-        description:
-            - Gitlab password for login_user
-    login_token:
-        description:
-            - Gitlab token for logging in.
-    name:
-        description:
-            - Name of the group you want to create.
-        required: true
-    path:
-        description:
-            - The path of the group you want to create, this will be server_url/group_path
-            - If not supplied, the group_name will be used.
+  server_url:
+    description:
+      - Url of Gitlab server, with protocol (http or https).
```

Comment:
Same remarks here, use URL instead of Url.
```suggestion
      - The URL of the Gitlab server, with protocol (i.e. http or https).
```

Reply:
Will be fix in the next commit